config/packages/security.yaml

```diff
@@ -1,13 +1,30 @@
 security:
+    encoders:
```

Comment:
This is needed only for `lexik_jwt_authentication.jwt_token_authenticator` to don't hit `SignInCommand` each request as it doesn't make sense